build.assets/version-ops/cmd/check/main.go

```diff
@@ -24,15 +24,13 @@ import (
 	"context"
 	"flag"
 	"log"
-	"sort"
 	"strings"
 	"time"
 
 	"golang.org/x/mod/semver"
 
+	"github.com/gravitational/teleport/build.assets/version-ops/lib/github"
```

Comment:
I've started a  module in `build.assets/tooling` for build-related tooling (basically anything used from the Makefile down). That may (or may not be) the right place for this.

Reply:
Moved to `build.assets/tooling` in 1605a41.